sql/item_mods.sql

```diff
@@ -24570,8 +24574,544 @@ INSERT INTO `item_mods` VALUES (21460,5,5);
+INSERT INTO `item_mods` VALUES (21706,757,5);
+INSERT INTO `item_mods` VALUES (21706,758,5);
+INSERT INTO `item_mods` VALUES (21706,759,5);
+INSERT INTO `item_mods` VALUES (21706,760,5);
+INSERT INTO `item_mods` VALUES (21706,761,5);
+INSERT INTO `item_mods` VALUES (21706,762,5);
+INSERT INTO `item_mods` VALUES (21706,763,5);
+INSERT INTO `item_mods` VALUES (21706,764,5);
+INSERT INTO `item_mods` VALUES (21706,765,5);
+INSERT INTO `item_mods` VALUES (21706,766,5);
+INSERT INTO `item_mods` VALUES (21706,767,5);
+INSERT INTO `item_mods` VALUES (21706,768,5);
+INSERT INTO `item_mods` VALUES (21706,769,5);
+INSERT INTO `item_mods` VALUES (21706,770,5);
+INSERT INTO `item_mods` VALUES (21706,771,5);
+INSERT INTO `item_mods` VALUES (21706,772,5);
+INSERT INTO `item_mods` VALUES (21706,773,5);
+INSERT INTO `item_mods` VALUES (21706,774,5);
+INSERT INTO `item_mods` VALUES (21706,775,5);
+INSERT INTO `item_mods` VALUES (21706,776,5);
+INSERT INTO `item_mods` VALUES (21706,777,5);
+INSERT INTO `item_mods` VALUES (21706,778,5);
+INSERT INTO `item_mods` VALUES (21706,779,5);
+INSERT INTO `item_mods` VALUES (21706,780,5);
+INSERT INTO `item_mods` VALUES (21706,781,5);
+INSERT INTO `item_mods` VALUES (21706,782,5);
+INSERT INTO `item_mods` VALUES (21706,783,5);
+INSERT INTO `item_mods` VALUES (21706,784,5);
+INSERT INTO `item_mods` VALUES (21706,785,5);
+INSERT INTO `item_mods` VALUES (21706,786,5);
+INSERT INTO `item_mods` VALUES (21706,787,5);
+INSERT INTO `item_mods` VALUES (21706,788,5);
+INSERT INTO `item_mods` VALUES (21706,789,5);
+INSERT INTO `item_mods` VALUES (21706,790,5);
+INSERT INTO `item_mods` VALUES (21706,791,5);
+INSERT INTO `item_mods` VALUES (21706,792,5);
+INSERT INTO `item_mods` VALUES (21706,793,5);
+INSERT INTO `item_mods` VALUES (21706,794,5);
+INSERT INTO `item_mods` VALUES (21706,795,5);
+INSERT INTO `item_mods` VALUES (21706,796,5);
+INSERT INTO `item_mods` VALUES (21706,797,5);
+INSERT INTO `item_mods` VALUES (21706,798,5);
+INSERT INTO `item_mods` VALUES (21706,799,5);
+INSERT INTO `item_mods` VALUES (21706,800,5);
+INSERT INTO `item_mods` VALUES (21706,801,5);
+INSERT INTO `item_mods` VALUES (21706,802,5);
+INSERT INTO `item_mods` VALUES (21706,803,5);
+INSERT INTO `item_mods` VALUES (21706,804,5);
+INSERT INTO `item_mods` VALUES (21706,805,5);
+INSERT INTO `item_mods` VALUES (21706,806,5);
+INSERT INTO `item_mods` VALUES (21706,807,5);
+INSERT INTO `item_mods` VALUES (21706,808,5);
+INSERT INTO `item_mods` VALUES (21706,809,5);
+INSERT INTO `item_mods` VALUES (21706,810,5);
+INSERT INTO `item_mods` VALUES (21706,811,5);
+INSERT INTO `item_mods` VALUES (21706,812,5);
+INSERT INTO `item_mods` VALUES (21706,813,5);
+INSERT INTO `item_mods` VALUES (21706,814,5);
+INSERT INTO `item_mods` VALUES (21706,815,5);
+INSERT INTO `item_mods` VALUES (21706,816,5);
+INSERT INTO `item_mods` VALUES (21706,817,5);
+INSERT INTO `item_mods` VALUES (21706,818,5);
+INSERT INTO `item_mods` VALUES (21706,819,5);
```

Comment:
@Fizztastic `MOD_ALL_WSDMG_ALL_HITS` and `MOD_ALL_WSDMG_FIRST_HIT` now exist if you rebase onto latest master. Neither wiki states which items use which (the in game helptext is the same) so I'd assume an item uses MOD_ALL_WSDMG_ALL_HITS until proven otherwise.